code/__DEFINES/shuttles.dm

```diff
@@ -87,3 +87,10 @@
 ///Check for arena shuttle, if the bubblegum has died this round
 GLOBAL_VAR_INIT(bubblegum_dead, FALSE)
 
+// Alert level related for new thing im porting --- cowbot93
+#define ALERT_COEFF_AUTOEVAC_NORMAL 2.5
```

Comment:
Check to ensure these are even the same ratios that we actually use

Reply:
adjusted to same ratios in next push